packages/contracts/test/voting/token-voting-setup.ts

```diff
@@ -6,51 +6,37 @@ import {ERC20, TokenVotingSetup} from '../../typechain';
 import {customError} from '../test-utils/custom-error-helper';
 import {deployNewDAO} from '../test-utils/dao';
 import {getInterfaceID} from '../test-utils/interfaces';
-import {ONE_HOUR} from '../test-utils/voting';
-
-enum Op {
-  Grant,
-  Revoke,
-  Freeze,
-  GrantWithOracle,
-}
+import {Operation} from '../core/permission/permission-manager';
+import {
+  VotingSettings,
+  VotingMode,
+  pct16,
+  ONE_HOUR,
+} from '../test-utils/voting';
+
+let defaultData: any;
+let defaultVotingSettings: VotingSettings;
+let defaultVokenSettings: {addr: string; name: string; symbol: string};
```

Comment:
i assume u mean: defaultTokenSettings

Reply:
haha, yes indeed
Fixed in 8b2bd9a52cb2f8d3aa7a6d5a8e89317b5f93003d